torchaudio/backend/sox_io_backend.py

```diff
@@ -10,6 +10,7 @@
 from .common import AudioMetaData
 
 
+@_mod_utils.requires_sox()
 @_mod_utils.requires_module('torchaudio._torchaudio')
```

Comment:
I think the logic of `requires_sox` contains that of `requires_module('torchaudio._torchaudio')`, so we should be able to replace `'torchaudio._torchaudio'`.